dev_tools/nbformat

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Formats ipython notebooks with tensorflow-docs nbformat tool.
+#
+# Usage:
+#     check/nbformat [--apply]
+#
+# Without '--apply', the diff that would be applied is printed and the exit
+# status is 1 if there are any changes or else 0 if no changes are needed.
+#
+# With '--apply', the exit status is 0 and the changed files are actually
+# reformatted.
+
+only_print=1
+for arg in "$@"; do
+    if [[ "${arg}" == "--apply" ]]; then
+        only_print=0
+    else
+        echo -e "\033[31mToo many arguments. Expected [--apply].\033[0m" >&2
+        exit 1
+    fi
+done
+
+# Get the working directory to the repo root.
+thisdir="$(dirname "${BASH_SOURCE[0]}")" || exit $?
+topdir="$(git -C "${thisdir}" rev-parse --show-toplevel)" || exit $?
+cd "${topdir}" || exit $?
+
+# check if the nbfmt tool in tensorflow-docs is installed and working
+if ! python3 -c "import tensorflow_docs.tools.nbfmt"; then
+    echo -e "\033[31mMissing installation of tensorflow-docs Python package.\033[0m"
+    echo -e "\033[31mPlease do 'pip install -r dev_tools/requirements/notebook.env.txt'.\033[0m"
```

Comment:
The previous wording of the error message from https://github.com/quantumlib/Cirq/pull/7004 might be more informative.  The problem fixed there was that the tensorflow-docs package _was installed_, but failed to import because of incompatibility of protobuf versions.

Reply:
> The previous wording of the error message from [quantumlib/Cirq#7004](https://github.com/quantumlib/Cirq/pull/7004) might be more informative. The problem fixed there was that the tensorflow-docs package _was installed_, but failed to import because of incompatibility of protobuf versions.

I personally found the message _less_ informative, or else I wouldn't have taken time to try to improve it. 

But your point is taken: the `import` can fail for reasons other than the tensorflow-docs package not being installed, so the new message may not always be correct.

It's unfortunate that improving this would take too much time. I guess I'll revert the message.